Dockerfile

```diff
@@ -1,2 +1,2 @@
-FROM registry.opensuse.org/yast/head/containers/yast-ruby:latest
+FROM yastdevel/ruby:sle15-sp1
```

Comment:
@lslezak SP1 uses latest?

Reply:
For SP1 you should use `yastdevel/ruby:sle15-sp1`, that's correct.

Reply:
Ah, this is for master, for master use `registry.opensuse.org/yast/head/containers/yast-ruby:latest`.